website/docs/r/one_dashboard.html.markdown

```diff
@@ -260,6 +338,27 @@ The following arguments are supported:
   * `title` - (Optional) A human-friendly display string for this value.
   * `value` - (Required) A possible variable value
 
+### Nested `Null Values` blocks
+
+The following arguments are supported:
+
+* `null_value` -  Choose an option in displaying null values. Accepted values are `deafult`, `remove`, `preserve`, or `zero`.
```

Comment:
`deafult`

Reply:
@kidk do you want me to add the default value for this attribute ?

Reply:
updated.